### PR TITLE
minor updates on the try in client page

### PR DESCRIPTION
**Pull request by antoine-richard**

### CODE REVIEW

File: modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+* your whole API definition : a project is created in Restlet Client with pre-defined requests for each of the API’s operations. You will then be able to set up complex scenarios that emulate real-life usages of your API.
 * a single operation : Restlet Client is opened with a ready-to-go request *for the selected* API operation
 
-
-For both of these purposes, Restlet Studio includes a "Try in Client" button, both in the editor view, and published documentation.
-
-If you have not yet installed Restlet Client, you will be redirected to the chrome store to install it. Note that once you have a Restlet account, you automatically have access to all modules, including Restlet Client and Restlet Cloud, with the same account.
+If you have not yet installed Restlet Client, you will be redirected to the Google Web Store to install it. Note that once you have a Restlet account, you automatically have access to all modules, including Restlet Client and Restlet Cloud, with the same account.
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md]: ptet qu'on peut virer la deuxième phrase sur les accounts? j'ai l'impression qu'elle est arrivée là par erreur (elle y étais déjà avant ta PR)

File: modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md [hunk 2]
```diff
@@ -39,9 +38,9 @@ The following table gives you details about the data imported into Restlet Clien
 
 | Restlet Studio item | Corresponding Restlet Client item | Comments
 |---------------------|-----------------------------------|---------
-| API | root project | Single project, which name is given by the name of the API in Restlet Studio.
-| Section | services |
-| Operation | requests | Each operation of each resource located inside the same section are gathered under the same service. Path variables, query parameters and headers are imported too.
+| API | Root Project | Single project, which name is given by the name of the API in Restlet Studio.
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md]: "whose name is based on the name of the API in Restlet Studio"

File: modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md [hunk 3]
```diff
@@ -39,9 +38,9 @@ The following table gives you details about the data imported into Restlet Clien
...
+| Section | Service |
+| Operation | Request | Each operation of each resource located inside the same section are gathered under the same service. Path variables, query parameters and headers are imported too.
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md]: Can simplify (don't mention resource?), plus there is a plural/singular error:

"All the operations from a Restlet Studio **Section** are gathered under the same Restlet Client **Service**. ...

File: modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md [hunk 4]
```diff
@@ -39,9 +38,9 @@ The following table gives you details about the data imported into Restlet Clien
...
 
-When you import another time an API from Studio, elements are matched thanks to their name so that the existing items in Client are overwritten (except the assertions, which are specific to Client), but no Client items are deleted.
+When you import another time an API from Studio, elements are matched thanks to their name so that the existing items in Client are updated (except the assertions, which are kept untouched - they are specific to Client), but no items are deleted in Restlet Client.
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md]: "When you export an entire API from Studio to Client more than once, ... "

Et mettre un point avant "but":

No items are deleted in ...

File: modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md (hunk 1 above)
Comment (antoine-richard) [reply modules/com.restlet/tech-doc/studio/guide/06_platform/01_tryinclient.md]: yes

**Pull request merged**